Assets/Scripts/UI/StatBar.cs

```diff
@@ -0,0 +1,93 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEditor;
+using UnityEngine;
+using UnityEngine.UI;
+
+public class StatBar : MonoBehaviour
+{
+    public string SpriteLocation = "Sprites/PlayerHungerStates_Sprite";
```

Comment:
Please use:
`[SerializedField] private Sprite[] playerHungerStateSprites;`

Reply:
FYI: You can use this for multiple references:
https://gyazo.com/4fea78d861640bbb93918df96a4dc0d6

Reply:
serialized, unity editor is annoying, took a while until I found out that you can lock panels